site/_posts/2017-08-11-back-to-basics.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Last Week in ManageIQ: Back to Basics"
+author: jntullo
+date: 2017-08-15
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello there! It's [Jillian](https://github.com/jntullo) again, here with the latest happenings the last *two* weeks in ManageIQ. 
+I probably cannot top last week's [edition](http://manageiq.org/blog/2017/08/limericks-week-in-miq/), so I've decided to
+bring it back to basics for this one. Read on to see what's been happening in our world over the past two weeks. 
+
+## Featured
+
+If you're users of or contributors to the API, you may be interested to hear that the API now has its own repository!
+Head on over to its new location [here](https://github.com/ManageIQ/manageiq-api) and check out the [talk](http://talk.manageiq.org/t/proposed-split-repo-manageiq-api/2587/3) 
+article for more information.
+
+## Improved
+
+### Zone Selection
+Zone selection has been improved in the UI to include [all zones](https://github.com/ManageIQ/manageiq-ui-classic/pull/1769), not just the zones in the current region. 
+
+### Update UI without Database Connection
+The task of updating the UI did not need a database connection to execute, but was still failing without one. An [update to that task](https://github.com/ManageIQ/manageiq-ui-classic/pull/1887)
+will allow `rake update:ui` to run without a database connection. 
+
+## Fixed
+
+### Removed assumptions from tests 
+Making assumptions in tests can lead to sporadic failures. By removing the assumption that the [region is 99](https://github.com/ManageIQ/manageiq/pull/15731) in tests, we improve our test reliability. 
```

Comment:
I think this should say "that the region is not 99" (since it used to be 1, but is now random)